### DISCUSSION TITLE
is it possible to step in a line that has a highlighted difference in history comparisons?

### DISCUSSION THREAD

**Discussion by istinnstudio** [apache/netbeans#6545] [Q&A] [answered]
hi... when comparing source and history versions, is it possible to step in the line that has a difference highlighted? Is there a shortcut or a function that can do direct cursor placement to a line that has a difference? It seems lines are being highlighted but in code with thousands lines is very difficult to move at difference positions. Maybe I am missing something or is this just a missing feature? I guess the cursor position should be able to move easily just like the "find" function, with previous and next button or just a key shortcut. thanks

**Comment by JoachimRohde** [comment]
![prevNextDiff](https://github.com/apache/netbeans/assets/1743626/c93f5723-f8ed-4c77-8a4e-26b7ea90ccbd)
You can navigate through the differences with the arrow buttons (red circle).

**Reply by istinnstudio** [reply]
Actually I mean the difference inside the same comparison, the current 2 files loaded. Not the next or previous file. I mean the next previous "change" in the same file. There are cases (especially in large classes with thousands lines of code) that the changes could be many and in various places very difficult to track when comparing.

**Comment by JoachimRohde** [comment, marked as answer]
The arrows show the previous/next difference within this comparison. It does not load the next file. Give it a try

**Comment by istinnstudio** [comment]
wow, you are absolutely right, arrows behavior was misleading for me because with cases with only one difference, arrow moved to the next raw in history list,  so I assumed this is the default behavior which is not. Thanks for clarifying that out.
